integration-tests/e2e/kubetest/description/1.19/skip.json

```diff
@@ -493,182 +493,5 @@
-  { "testcase": "[sig-storage] Zone Support Verify PVC creation with compatible policy and datastore without any zones specified in the storage class fails (No shared datastores exist among all the nodes)"},
-  { "testcase": "[k8s.io] [sig-node] NodeProblemDetector [DisabledForLargeClusters] should run without error"},
-  { "testcase": "[sig-storage] Zone Support Verify a pod is created and attached to a dynamically created PV, based on allowed zones specified in storage class "},
-  { "testcase": "[sig-storage] Zone Support Verify dynamically created pv with allowed zones specified in storage class, shows the right zone information on its labels"},
-  { "testcase": "[sig-storage] Zone Support Verify PVC creation with an invalid VSAN capability along with a compatible zone combination specified in storage class fails"},
-  { "testcase": "[sig-storage] Pod Disks schedule pods each with a PD, delete pod and verify detach [Slow] for read-only PD with pod delete grace period of \"default (30s)\""},
-  { "testcase": "[sig-storage] Zone Support Verify PVC creation fails if the availability zone specified in the storage class have no shared datastores under it."},
-  { "testcase": "[sig-storage] Zone Support Verify a pod is created and attached to a dynamically created PV, based on multiple zones specified in storage class "},
-  { "testcase": "[sig-network] Services should implement service.kubernetes.io/service-proxy-name"},
-  { "testcase": "[sig-storage] Zone Support Verify a pod is created and attached to a dynamically created PV, based on multiple zones specified in the storage class. (No shared datastores exist among both zones)"},
-  { "testcase": "[sig-storage] Zone Support Verify PVC creation with incompatible storagePolicy and zone combination specified in storage class fails"},
-  { "testcase": "[sig-storage] PersistentVolumes-local  [Volume type: dir] Set fsGroup for local volume should set different fsGroup for second pod if first pod is deleted"},
-  { "testcase": "[sig-api-machinery] AdmissionWebhook Should honor timeout"},
-  { "testcase": "[sig-storage] Pod Disks schedule pods each with a PD, delete pod and verify detach [Slow] for read-only PD with pod delete grace period of \"immediate (0s)\""},
-  { "testcase": "[sig-storage] Zone Support Verify a pod is created on a non-Workspace zone and attached to a dynamically created PV, based on the allowed zones and storage policy specified in storage class"},
-  { "testcase": "[sig-storage] PersistentVolumes-local  [Volume type: dir-link] Set fsGroup for local volume should set different fsGroup for second pod if first pod is deleted"},
-  { "testcase": "[sig-storage] EmptyDir wrapper volumes should not cause race condition when used for git_repo [Serial] [Slow]", "description": "Fails on GL (not investigated why), but test looks for a race condition in K8S which was fixed in K8S v1.9.x and uses GitRepo type volumes which are deprecated since K8S v1.11.0" },
```

Comment:
I'm not sure why this line (and the next one) were removed from the skip-list? Did you really try to rerun them and they now work?
(also they are not on the working.json, so probably you deleted them by accident?)

Reply:
@dguendisch I ran the tests in was, gcp, openstack with the following command `go run -mod=vendor ./integration-tests/e2e --debug=true --k8sVersion=1.19.2 --cloudprovider=<cloudprovider> --testcasegroup="conformance"`

The tests ran fine. Yes, I accidentally deleted two test cases in working.json and forgot to move them into skip.json. I updated the PR.

Reply:
Hmm, `testcasegroup="conformance"` will however invoke the conformance tests, not the tests that were moved in this PR!
I'll test it out myself, hopefully next week and can give then better feedback.